modules/git/repo_commit_nogogit.go

```diff
@@ -108,7 +108,7 @@ func (repo *Repository) getCommitFromBatchReader(rd *bufio.Reader, id SHA1) (*Co
 			return nil, err
 		}
 
-		commit.CommitMessage = strings.TrimSpace(tag.Message)
+		commit.CommitMessage = tag.Message
 		commit.Author = tag.Tagger
 		commit.Signature = tag.Signature
```

Comment:
Why are we overwriting the commit's data here with incorrect data from the tag? I think this is the root of the problem and I guess if we just remove these three lines, the issue may be fixed.